spec/models/timeline_spec.rb

```diff
@@ -120,31 +120,38 @@
     let(:date) { 1.day.ago.strftime("%F") }
     let(:name) { "reopenedDate" }
     let(:event_param) { [name, date] }
+    subject { Timeline::ComplaintEvent.new(event_param) }
```

Comment:
[thought] doesn't need to be in scope with this issue, but i think the individual Timeline Events have maybe outgrown the original file, and should be housed in their own files!